docs/qa-guide.md

```diff
@@ -83,23 +85,29 @@ exit
 ```
 nix-build -A ghcjs.reflex-todomvc
 ```
-If successful this will give you a file path. Navigate to a browser (like chrome) and enter the following (with the path) into the bar:
-```
-file:///<insert nix path from previous instruction here>/bin/reflex-todomvc.jsexe/index.html
-```
+If successful this will give you a file path. Run the following command
+depending on your system:
```

Comment:
Let's avoid manual wordwrapping in these docs simply because it makes diffs harder.

Reply:
My editor added them without me noticing. I have reverted, feel free to squash when merging